src/shared/telemetries/telemetrysender/global_sender.go

```diff
@@ -0,0 +1,26 @@
+package telemetrysender
+
+import (
+	"github.com/otterize/intents-operator/src/shared/telemetries/telemetriesgql"
+	"github.com/sirupsen/logrus"
+	"sync"
+)
+
+var (
+	stdOnce         = sync.Once{}
+	std             *TelemetrySender
```

Comment:
```suggestion
	sender             *TelemetrySender
```

Reply:
std in logrus refers to the StandardLogger - here it's the telemetry sender, so we should name it something appropriate